packages/shared/src/components/FocusBlurHandler.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  children: PropTypes.node,
+  onFocus: PropTypes.func,
+  onBlur: PropTypes.func,
+};
+
+const defaultProps = {
+  children: null,
+  onFocus: null,
+  onBlur: null,
+};
+
+// This component wraps its children in an <a /> tag which is the most reliable way to
+// support tabIndex focusing accross browsers for SVG < v2.0
+export default class FocusBlurHandler extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.handleOnClick = this.handleOnClick.bind(this);
+  }
+
+  handleOnClick(e) { // eslint-disable-line class-methods-use-this
+    e.preventDefault();
+  }
+
+  render() {
+    const {
+      children,
+      onFocus,
+      onBlur,
+    } = this.props;
+
+    return (
+      <a // eslint-disable-line jsx-a11y/no-static-element-interactions
```

Comment:
what you'd likely want to use instead of an `a` if you wanna go with this approach is to use a `button`

Reply:
@backwardok [a `button` is not a valid `svg` child, `a` is](https://www.w3.org/TR/SVG/struct.html#SVGElement). 

as mentioned above, this approach seemed like the recommendation according to this [Stack Overflow answer](https://stackoverflow.com/a/28654521/1933266) and [this section of a11yjs.io](https://allyjs.io/tutorials/focusing-in-svg.html)

Reply:
Dang it :( That seems reasonable then

Reply:
maybe add role="button"?